app/main.py

```diff
@@ -1,6 +1,38 @@
+import os
+
+import requests
+
+
+# -----Constants---------------------------------------------------------------
+API_KEY = os.getenv("API_KEY")
```

Comment:
Ensure that the `API_KEY` environment variable is set before making the API request. Consider adding a check to handle cases where `API_KEY` might be `None` to avoid potential runtime errors.